sql/item.cc

```diff
@@ -3615,7 +3618,9 @@ bool Item_field::eq(const Item *item, bool binary_cmp) const
 
 table_map Item_field::used_tables() const
 {
-  if (field->table->const_table)
+  if (!field ||
+      !field->table ||
+      field->table->const_table)
```

Comment:
Due to usage of used_tables() in setup_fields(), we can now end up calling this function before the item is declared fixed, and field is still nullptr.